template/.eslintrc.js

```diff
@@ -1,4 +1,8 @@
 module.exports = {
   root: true,
   extends: ['plugin:echobind/react-native'],
+  rules: {
```

Comment:
These were both causing issues for me, not sure if we should add them to the plugin or leave them here